toonz/sources/stopmotion/stopmotioncontroller.cpp

```diff
@@ -1374,32 +1375,50 @@ StopMotionController::StopMotionController(QWidget *parent) : QWidget(parent) {
     // Make Options Page
     QGroupBox *webcamBox  = new QGroupBox(tr("Webcam Options"), this);
     QGroupBox *dslrBox    = new QGroupBox(tr("DSLR Options"), this);
-    QGroupBox *timerFrame = new QGroupBox(tr("Time Lapse"), this);
-    m_timerCB             = new QCheckBox(tr("Use time lapse"), this);
-    m_timerIntervalFld    = new DVGui::IntField(this);
-    timerFrame->setObjectName("CleanupSettingsFrame");
+    m_timerCB             = new QGroupBox(tr("Use Time Lapse"), this);
+    m_timerIntervalFld    = new DVGui::DoubleField(this, true, 1);
+    m_timerCB->setCheckable(true);
+    m_timerCB->setObjectName("CleanupSettingsFrame");
     m_timerCB->setChecked(false);
-    m_timerIntervalFld->setRange(0, 60);
-    m_timerIntervalFld->setValue(10);
-    m_timerIntervalFld->setDisabled(true);
+    m_timerIntervalFld->setRange(0.0, 60.0);
+    m_timerIntervalFld->setValue(0.0);
 
-    m_postCaptureReviewFld = new DVGui::IntField(this);
-    m_postCaptureReviewFld->setRange(0, 10);
+    m_postCaptureReviewFld = new DVGui::DoubleField(this, true, 1);
+    m_postCaptureReviewFld->setRange(0.0, 10.0);
+    m_postCaptureReviewFld->setValue(0.0);
 
     m_subsamplingFld = new DVGui::IntField(this);
     m_subsamplingFld->setRange(1, 30);
     m_subsamplingFld->setDisabled(true);
 
-    m_placeOnXSheetCB = new QCheckBox(this);
+    m_placeOnXSheetCB = new QCheckBox(tr("Place on XSheet"), this);
     m_placeOnXSheetCB->setToolTip(tr("Place the frame in the Scene"));
 
-    m_directShowLabel = new QLabel(tr("Use Direct Show Webcam Drivers"), this);
```

Comment:
@justburner There is reference to this variable on line 2052 which is only used on non Windows builds. 

```
#ifndef _WIN32
  m_directShowCB->hide();
  m_directShowLabel->hide();
#endif
```

Since you don't create/initialize this label, it crashes when it tries to hide it.